### PR TITLE
Add Yandex.Cloud KMS wrapper

### PR DESCRIPTION
**Pull request by AndreyZamyslov**
These changes add support for [Yandex.Cloud](https://cloud.yandex.com/) [KMS](https://cloud.yandex.com/docs/kms/)

Could you please review the code?

### CODE REVIEW

**Comment by AndreyZamyslov** [comment]
@jefferai Could you please help with reviewing the changes?

File: wrappers/yandexcloudkms/yandexcloudkms.go [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+	if err != nil {
+		return nil, fmt.Errorf("error wrapping data: %w", err)
+	}
+
+	if k.client == nil {
+		return nil, fmt.Errorf("nil client")
+	}
+
+	encryptResponse, err := k.client.Encrypt(
+		context.Background(),
```
Comment (jefferai) [inline wrappers/yandexcloudkms/yandexcloudkms.go:150]: Don't use background -- use the context passed into this function.

File: wrappers/yandexcloudkms/yandexcloudkms.go [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+}
+
+// Decrypt is used to decrypt the ciphertext. This should be called after Init.
+func (k *Wrapper) Decrypt(_ context.Context, in *wrapping.EncryptedBlobInfo, aad []byte) (pt []byte, err error) {
+	if in == nil {
+		return nil, fmt.Errorf("given input for decryption is nil")
+	}
+
+	decryptResponse, err := k.client.Decrypt(
+		context.Background(),
```
Comment (jefferai) [inline wrappers/yandexcloudkms/yandexcloudkms.go:185]: Same comment

File: wrappers/yandexcloudkms/yandexcloudkms.go (hunk 1 above)
Comment (AndreyZamyslov) [reply wrappers/yandexcloudkms/yandexcloudkms.go]: Fixed

File: wrappers/yandexcloudkms/yandexcloudkms.go (hunk 2 above)
Comment (AndreyZamyslov) [reply wrappers/yandexcloudkms/yandexcloudkms.go]: Fixed

**Comment by jefferai** [comment]
Thanks!

**Comment by AndreyZamyslov** [comment]
Thank you!

**Pull request merged**